tests/integration_tests/clouds.py

```diff
@@ -31,6 +31,17 @@ def __init__(self, settings=integration_settings):
         self.cloud_instance = self._get_cloud_instance()
         self.image_id = self._get_initial_image()
 
+    def emit_settings_to_log(self) -> None:
+        log.info(
+            "\n".join(
+                ["Settings:"]
+                + [
+                    "{}={}".format(key, getattr(self.settings, key))
```

Comment:
Are there any sensitive settings we would be exposing here that should be redacted?

Reply:
The current set is:

```py
In [1]: from tests.integration_tests import integration_settings                                                                                                                                                                                                                          

In [2]: integration_settings.current_settings                                                                                                                                                                                                                                             
Out[2]: 
['KEEP_INSTANCE',
 'PLATFORM',
 'INSTANCE_TYPE',
 'OS_IMAGE',
 'EXISTING_INSTANCE_ID',
 'CLOUD_INIT_SOURCE',
 'GCE_PROJECT',
 'GCE_REGION',
 'GCE_ZONE',
 'OCI_COMPARTMENT_ID']
```

so no, I don't think so. Good thought, though, we'll need to be mindful of that going forward.